src/ansys/fluent/core/solver/flobject.py

```diff
@@ -212,7 +212,10 @@ class Textual(Property):
 
     def allowed_values(self):
         """Get the allowed values of the object."""
-        return self.get_attr("allowed-values", (list, str))
```

Comment:
@prmukherj regarding this old PR, this could be relevant to the testing issue task #1496.

Reply:
@prmukherj @mkundu1 @hpohekar @ajain-work this change has now been superseded by identical changes - closing PR